oidc-service-impl/src/main/java/io/mosip/esignet/services/OAuthServiceImpl.java

```diff
@@ -104,9 +103,18 @@ public TokenResponse getTokens(TokenRequest tokenRequest,boolean isV2) throws Es
 
         authenticateClient(tokenRequest, clientDetailDto,isV2);
 
+        Map<String,Object> additionalConfig= clientDetailDto.getAdditionalConfig();
+        String userinfoResponsetype = null;
+        if (additionalConfig != null && additionalConfig.containsKey("user_info_response_type")) {
+            Object responseTypeValue = additionalConfig.get("user_info_response_type");
+            if (responseTypeValue != null) {
+                userinfoResponsetype = (String) responseTypeValue;
+            }
+        }
+
```

Comment:
We might need the same logic in many other places, can we write a small utility method in the DTO class itself?